docs/reference/networking/internal-network-requirements.md

```diff
@@ -0,0 +1,51 @@
+(reference-internal-network-requirements)=
+# Internal network requirements
+
+Landscape consists of a number of individual services and dependencies. These components communicate with each other across the network in a variety of ways. Landscape Clients may also communicate with Landscape Server across the internet.
+
+This page documents how Landscape Server's components communicate with each other – which communication protocols are used and what network settings are required to allow this. For more information on the individual components and their purposes, see {ref}`this explanation of the Landscape Server architecture <explanation-server-architecture>`.
+
+![Landscape Network Diagram](/assets/images/landscape-server-networking.jpg "Landscape Networking")
+
+## External network traffic
+
+External clients communicate with Landscape Server over the internet. These clients include web browsers interacting the Landscape's web UI, Landscape API clients, Landscape-managed instances, and Landscape repository package uploaders.
+
+The first point of contact for all of this traffic is the reverse proxy, which also acts as a load-balancer and performs TLS termination. In most Landscape Server deployments, this role is filled by [HAProxy](https://www.haproxy.org/), but in some deployments, it may be [Apache Server](https://httpd.apache.org/).
+
+All External network traffic to and from Landscape uses HTTP. Consequently, TCP ports 80 and 443 (for HTTPS) on the reverse proxy must be exposed to the wide area network. In addition, port 6554 must be exposed for gRPC (HTTP/2) communication with [Ubuntu Pro for WSL](https://documentation.ubuntu.com/wsl/stable/).
```

Comment:
```suggestion
All external network traffic to and from Landscape uses HTTP. Consequently, TCP ports 80 and 443 (for HTTPS) on the reverse proxy must be exposed to the wide area network. In addition, port 6554 must be exposed for gRPC (HTTP/2) communication with [Ubuntu Pro for WSL](https://documentation.ubuntu.com/wsl/stable/).
```
Fixed typo, but also - this feels worded odd, but I'm not sure how it should be changed. The first sentence says everything uses HTTP, and then the next one says to expose those ports for HTTPS. Is that first one supposed to be HTTPS instead..? Or are you saying Landscape needs to be set up for both HTTP and HTTPS?